README.md

```diff
@@ -264,5 +264,36 @@ Golang agent supports the following dynamic configurations.
 |:-----------------:|:----------------------------------------------------------------------------------------:|:--------------------:|
 | agent.sample_rate | The percentage of trace when sampling. It's `[0, 1]`, Same with `WithSampler` parameter. |         0.1          |
 
+## Process
+
+This feature is used in cooperation with the [skywalking-rover](https://github.com/apache/skywalking-rover) project.
```

Comment:
I think you missed one, I suggest this feature(temp files) should be optional and OFF on default.

Reply:
It is recommended to enable this feature through environment variable

Reply:
Updated and let the process label and process status hook enabled could update through the environment variable.